appservice/src/tree/AppSettingsTreeItem.ts

```diff
@@ -33,27 +33,16 @@ export function validateAppSettingKey(settings: StringDictionary, client: SiteCl
     return undefined;
 }
 
-export class AppSettingsTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
-    public static contextValue: string = 'applicationSettings';
-    public readonly label: string = 'Application Settings';
-    public readonly childTypeLabel: string = 'App Setting';
-    public readonly contextValue: string = AppSettingsTreeItem.contextValue;
-    private _settings: StringDictionary | undefined;
+export class AppSettingsTreeItem extends AppSettingsTreeItemBase {
 
-    public get id(): string {
-        return 'configuration';
-    }
-
-    public get iconPath(): TreeItemIconPath {
-        return getThemedIconPath('settings');
-    }
+    public readonly _client: SiteClient;
 
-    public hasMoreChildrenImpl(): boolean {
-        return false;
+    constructor(parent: AzureParentTreeItem<ISiteTreeRoot>) {
+        super(parent, parent.root.client);
     }
 
-    public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzureTreeItem<ISiteTreeRoot>[]> {
-        this._settings = await this.root.client.listApplicationSettings();
+    public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzExtTreeItem[]> {
```

Comment:
I would think you need this in the base class. Is `loadMoreChildrenImpl` different for trial apps for some reason? Same with createChildImpl